### ISSUE TITLE
Failed! Something went wrong. Could not add the playlist to plex. Please try again. 

### ISSUE THREAD

**Issue by ChiefMedicalOfficer** [banjo/plex-web#1]
Installation seems to have went well without errors. 

When adding a new playlist I am met with the above error. I have tried 5 different playlists and all my Plex sections.
Output from terminal if it helps any. I installed it on both my Windows 10 machines resulting in the same issue.
```
127.0.0.1 - - [19/Jul/2019 20:02:41] "GET /connect HTTP/1.1" 200 -
127.0.0.1 - - [19/Jul/2019 20:03:14] "POST /connect HTTP/1.1" 302 -
127.0.0.1 - - [19/Jul/2019 20:03:15] "GET / HTTP/1.1" 200 -
127.0.0.1 - - [19/Jul/2019 20:03:20] "GET /addplaylist HTTP/1.1" 200 -
127.0.0.1 - - [19/Jul/2019 20:03:49] "POST /addplaylisttoplex HTTP/1.1" 200 -
127.0.0.1 - - [19/Jul/2019 20:03:56] "POST /addplaylisttoplex HTTP/1.1" 200 -
127.0.0.1 - - [19/Jul/2019 20:04:02] "POST /addplaylisttoplex HTTP/1.1" 200 -
127.0.0.1 - - [19/Jul/2019 20:04:09] "POST /addplaylisttoplex HTTP/1.1" 200 -
127.0.0.1 - - [19/Jul/2019 20:04:29] "POST /addplaylisttoplex HTTP/1.1" 200 -
127.0.0.1 - - [19/Jul/2019 20:06:24] "POST /addplaylisttoplex HTTP/1.1" 200 -
```

Windows 10
Python 3.7.4

Many thanks

**Comment by banjo** [comment]
Can you send me a picture of how the input forms look?

I haven't added any problem checking yet, but some reasons it might happen:
* Wrong ID
* Choosing a user that isn't connected to the server

I'll try to look into it further after the weekend.

**Comment by ChiefMedicalOfficer** [comment]
I'll send a pic when I get back on. 

I tried no user (I take it that defaults to me as the server owner), all users, some users and a user I knew was on at that moment.

I intentionally tried an incorrect token just to eliminate that too. After trying the correct one obviously.

Thanks again.

**Comment by acavataio** [comment]
I am seeing the same issue but I am in front of the computer, so let me know if I can help.  I was mainly trying it with no user, but also had no luck when adding them.  By help I mean troublehsoot with you, I am not capable of fixing it as I am not a dev, just a guy who likes the idea of this app.

**Comment by ChiefMedicalOfficer** [comment]
I've tried as many combinations of the options as I can.

![Screenshot](https://i.imgur.com/JBrNfyK.png)

**Comment by ChiefMedicalOfficer** [comment]
> 
> 
> I am seeing the same issue but I am in front of the computer, so let me know if I can help. I was mainly trying it with no user, but also had no luck when adding them. By help I mean troublehsoot with you, I am not capable of fixing it as I am not a dev, just a guy who likes the idea of this app.

I love the idea of this too. Would really help with curation of playlists especially for my older users.

**Comment by banjo** [comment]
I have added some basic error checking in the new branch" error-checking-to-adding-playlist". Only for wrong ID and user that doesn't have access. The error gets printed in the terminal. Download the new branch and try again and copy the new error message and I'll look at it!

**Comment by ChiefMedicalOfficer** [comment]
Excellent. I'll get that done later. Thanks.

**Comment by ChiefMedicalOfficer** [comment]
OK here we are. I realised looking at the errors that `lxml` seemed to be missing so I `pip install lxml` and a new Playlist has been created in Plex and populated with movies. This is great.

~~I am getting an infinite spinning wheel but I seem to be having some issues with command completing across other other programs so may not be related.~~ There's a couple of other small issues but I'll check them out a bit more before raising an issue.

I just had a look through the requirements and noticed lxml was added in. I didn't even notice that had been changed so didn't run through the setup again.

Thanks again.

![Image of Yaktocat](https://i.imgur.com/Ito07dE.pngg)
